src/replica/replica_stub.cpp

```diff
@@ -2766,6 +2768,19 @@ void replica_stub::on_group_bulk_load(group_bulk_load_rpc rpc)
         response.err = ERR_OBJECT_NOT_FOUND;
     }
 }
+// TODO: (Tangyanzhao) implement it later
+void replica_stub::on_detect_hotkey(detect_hotkey_rpc rpc)
+{
+    const auto &request = rpc.request();
+    auto &response = rpc.response();
+    replica_ptr rep = get_replica(request.pid);
+    if (rep != nullptr) {
+        response.err = ERR_OK;
+    } else {
+        response.err = ERR_SERVICE_NOT_FOUND;
```

Comment:
We usually use `ERR_OBJECT_NOT_FOUND` indicating can not found repica in this stub. Besides, add `derror_f` for this error, add replica pid infomation in err_hint.